src/features/bunnyTrove/components/EasterEgg.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useContext } from "react";
+
+import { MapPlacement } from "features/game/expansion/components/MapPlacement";
+import { PIXEL_SCALE } from "features/game/lib/constants";
+import { EasterHunt } from "features/game/types/game";
+import { ITEM_DETAILS } from "features/game/types/images";
+import { Context } from "features/game/GameProvider";
+
+export const EasterEgg: React.FC<Omit<EasterHunt, "generatedAt">> = ({
+  eggs,
+}) => {
+  const { gameService } = useContext(Context);
+  return (
+    <>
+      {eggs &&
+        eggs.map((egg, index) =>
+          !egg.collectedAt ? (
```

Comment:
TODO - check if eggs were generated in the last 12 hours.

This helps:

1. Hide stale data (i.e. just ticked over to midnight)
2. Hide eggs once event has finished